hack/tools/release/notes.go

```diff
@@ -512,12 +512,15 @@ func generateReleaseNoteEntry(c *commit) (*releaseNoteEntry, error) {
 		if strings.Contains(entry.title, "CAEP") || strings.Contains(entry.title, "proposal") {
 			entry.section = proposals
 		}
-	case strings.HasPrefix(entry.title, ":seedling:"), strings.HasPrefix(entry.title, "🌱"):
-		entry.section = other
-		entry.title = removePrefixes(entry.title, []string{":seedling:", "🌱"})
 	case strings.HasPrefix(entry.title, ":warning:"), strings.HasPrefix(entry.title, "⚠️"):
 		entry.section = warning
 		entry.title = removePrefixes(entry.title, []string{":warning:", "⚠️"})
+	case strings.HasPrefix(entry.title, "🚀"), strings.HasPrefix(entry.title, "🌱 Release v1."):
+		// Release trigger PRs from previous releases are not included in the release notes
+		return nil, nil
+	case strings.HasPrefix(entry.title, ":seedling:"), strings.HasPrefix(entry.title, "🌱"):
```

Comment:
Can we add a TODO to remove this once it's no longer relevant? If I understand right it should be unnecessary after the coming releases for 1.4.X and 1.5.X

Reply:
I'll add a todo
We can remove it one we release 1.6
Until then, these two commits will show up in the notes for all beta and RC releases for 1.6 (since these read from main)